ci/scripts/r_test.sh

```diff
@@ -15,13 +16,18 @@
 # specific language governing permissions and limitations
 # under the License.
 
-FROM golang:1.12
+set -ex
 
-COPY go/arrow/Gopkg.lock \
-     go/arrow/Gopkg.toml \
-     /arrow/go/arrow/
-WORKDIR /arrow/go/arrow
+: ${R_BIN:=R}
 
-RUN go get -d -t -v ./...
+source_dir=${1}/r
 
-CMD ["/bin/bash", "-c", "go install -v ./... && for d in $(go list ./... | grep -v vendor); do go test $d; done"]
+pushd ${source_dir}
+
+export LD_LIBRARY_PATH=${ARROW_HOME}/lib:${LD_LIBRARY_PATH}
+export R_LD_LIBRARY_PATH=${LD_LIBRARY_PATH}
+export _R_CHECK_FORCE_SUGGESTS_=false
```

Comment:
This is not necessary (I know this is copied from somewhere else) because r_deps.sh installs all suggested dependencies.